keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,556 @@
+    """Round number of filters based on depth multiplier."""
+    filters *= width_coefficient
+    minimum_depth = min_depth or depth_divisor
+    new_filters = max(
+        minimum_depth,
+        int(filters + depth_divisor / 2) // depth_divisor * depth_divisor,
+    )
+    return int(new_filters)
+
+
+def round_repeats(repeats, depth_coefficient):
+    """Round number of repeats based on depth multiplier."""
+    return int(math.ceil(depth_coefficient * repeats))
+
+
+def get_block_conv(args, activation, survival_probability, name):
+    # Determine which conv type to use:
+    if args["conv_type"] == "mb_conv":
+        return MBConvBlock(
+            input_filters=args["input_filters"],
+            output_filters=args["output_filters"],
+            expand_ratio=args["expand_ratio"],
+            kernel_size=args["kernel_size"],
+            strides=args["strides"],
+            se_ratio=args["se_ratio"],
+            activation=activation,
+            bn_momentum=0.9,
+            survival_probability=survival_probability,
+            name=name,
+        )
+    elif args["conv_type"] == "fused_mb_conv":
+        return FusedMBConvBlock(
+            input_filters=args["input_filters"],
+            output_filters=args["output_filters"],
+            expand_ratio=args["expand_ratio"],
+            kernel_size=args["kernel_size"],
+            strides=args["strides"],
+            se_ratio=args["se_ratio"],
+            activation=activation,
+            bn_momentum=0.9,
+            survival_probability=survival_probability,
+            name=name,
+        )
+    raise ValueError(
+        "Expected `block_args['conv_type']` to be "
+        "one of 'mb_conv', 'fused_mb_conv', but got "
+        f"`block_args['conv_type']={args['conv_type']}`"
+    )
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class EfficientNetV2Backbone(Backbone):
+    """Instantiates the EfficientNetV2 architecture.
+
+    Args:
+        include_rescaling: bool, whether to rescale the inputs. If set
+            to `True`, inputs will be passed through a `Rescaling(1/255.0)`
+            layer.
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        default_size: integer, default input image size.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        min_depth: integer, minimum number of filters.
+        activation: activation function.
+        blocks_args: list of dicts, parameters to construct block modules.
+        model_name: string, model name.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+            to use as image input for the model.
+    """
```

Comment:
An example call at the end would be very valuable here.

Reply:
Added